Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionCodeGenerator.cs

```diff
@@ -130,72 +157,25 @@ public static string GenerateWrapperCode(IEnumerable<InputActionMap> maps, IEnum
                     }
                 }
             }
-            writer.WriteLine("m_Initialized = true;");
             writer.EndBlock();
 
-            // Uninitialize method.
-            writer.WriteLine("private void Uninitialize()");
-            writer.BeginBlock();
-            foreach (var map in maps)
+            writer.WriteLine("public void Enable()");
+			writer.BeginBlock();
+            foreach (var set in maps)
             {
-                var mapName = CSharpCodeHelpers.MakeIdentifier(map.name);
-
-                if (options.generateInterfaces)
-                {
-                    var mapTypeName = CSharpCodeHelpers.MakeTypeName(map.name, "Actions");
-                    writer.WriteLine($"if (m_{mapTypeName}CallbackInterface != null)");
-                    writer.BeginBlock();
-                    writer.WriteLine($"{mapName}.SetCallbacks(null);");
-                    writer.EndBlock();
-                }
-
-                writer.WriteLine($"m_{mapName} = null;");
-
-                foreach (var action in map.actions)
-                {
-                    var actionName = CSharpCodeHelpers.MakeIdentifier(action.name);
-                    writer.WriteLine($"m_{mapName}_{actionName} = null;");
-
-                    if (options.generateEvents)
```

Comment:
Loosing this path I'm good with. Think it was never a good solution and `PlayerInput` has a more elegant version of it.

Reply:
Again, this is just the uninitalization part. We still generate the events if requested on initialization. I could remove that, though.

Reply:
Yup, nvm.